annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -504,6 +513,50 @@ private boolean hasAdminRights(Authentication authentication) {
 	}
     }
 
+    /**
+     * This method compares If-Match header with the current etag value.
+     * 
+     * @param etag    The current etag value
+     * @param request The request containing If-Match header
+     * @throws HttpException
+     */
+    public void checkIfMatchHeader(String etag, HttpServletRequest request) throws HttpException {
```

Comment:
the same methos exists in Entity API, please move implementation to API commons and also remove the method from Entity API

Reply:
moved to commons-api

Reply:
the method must be removed now from this class